libc/newhdrgen/yaml_to_classes.py

```diff
@@ -174,6 +174,7 @@ def add_function_to_yaml(yaml_file, function_details):
 
     with open(yaml_file, "r") as f:
         yaml_data = yaml.load(f, Loader=yaml.FullLoader)
+
```

Comment:
You can just remove this new line so that this PR doesn't change newhdrgen at all.

```suggestion
```